sonar-erlang-plugin/src/test/java/org/sonar/plugins/erlang/ErlangSquidSensorTest.java

```diff
@@ -58,13 +58,6 @@ public void setup() {
     when(metricFinder.<Integer>findByKey(CoreMetrics.STATEMENTS_KEY)).thenReturn(CoreMetrics.STATEMENTS);
     when(metricFinder.<Integer>findByKey(CoreMetrics.COMPLEXITY_KEY)).thenReturn(CoreMetrics.COMPLEXITY);
     when(metricFinder.<Integer>findByKey(CoreMetrics.COMMENT_LINES_KEY)).thenReturn(CoreMetrics.COMMENT_LINES);
-    when(metricFinder.<Integer>findByKey(CoreMetrics.PUBLIC_API_KEY)).thenReturn(CoreMetrics.PUBLIC_API);
```

Comment:
Is this completely removed from SonarQube?

Reply:
https://jira.sonarsource.com/browse/SONAR-8328 both deprecated and hidden, and aren't even in latest version docs, so I think it is safe to not test this.